cmd/world/cardinal/dev.go

```diff
@@ -83,6 +83,8 @@ var devCmd = &cobra.Command{
 			cmdErr <- err
 		}()
 
+		fmt.Println("Cardinal Started..")
```

Comment:
nit: "Starting Cardinal..."

Reply:
updated and move code before go routine func for cardinal cmd called